configs/milvus.yaml

```diff
@@ -121,10 +123,18 @@ pulsar:
 # If you want to enable kafka, needs to comment the pulsar configs
 # kafka:
 #   brokerList:
-#   saslUsername:
-#   saslPassword:
-#   saslMechanisms: PLAIN
-#   securityProtocol: SASL_SSL
```

Comment:
"securityProtocol" cannot be ignored, it can be one of the "SASL_PLAIN", "SSL", "SASL_SSL", according to different authentication ways.

Reply:
> "securityProtocol" cannot be ignored, it can be one of the "SASL_PLAIN", "SSL", "SASL_SSL", according to different authentication ways.

no, securityProtoco can be ignored, as we can know what kind of securityProtoco should be used according the sasl and ssl config

Reply:
Agreed on this part

Reply:
and it is actually clearer compared to what we have right now.

@chenraoCR would you mind rebase so we can bring in the minio part first?